test/sendwithus.js

```diff
@@ -295,3 +295,27 @@ module.exports.createTemplates = {
     });
   }
 };
+
+module.exports.renderTemplates = {
+    setUp: function (callback) {
+        this.sendwithus = sendwithusFactory(API_KEY);
+        this.data = {
```

Comment:
This should be more [aligned with the API](https://support.sendwithus.com/api/#renderapi) — several of those fields won't do anything. How about:

```js
this.data = {
    template: TEMPLATE,
    template_data: {
        name: 'name ' + Date().toString()
    },
    strict: false
}
```

Reply:
Thanks for the review and comments, I've made a few changes if you want to take another look.